doc/introduction/data.rst

```diff
@@ -83,25 +89,29 @@ function calls will change.
 >>> available_data.keys()
 dict_keys(['qspin', 'qchem'])
 >>> available_data['qchem'].keys()
-dict_keys(['HF', 'LiH', ...])
+dict_keys(['H2', 'LiH', ...])
 >>> available_data['qchem']['H2'].keys()
-dict_keys(['STO-3G'])
+dict_keys(['6-31G', 'STO-3G'])
 >>> print(available_data['qchem']['H2']['STO-3G'])
-['2.35', '1.75', '0.6', '1.85', ...]
+['0.5', '0.54', '0.62', '0.66', ...]
 
 Creating Custom Datasets
 ------------------------
 
 The functionality in :mod:`~pennylane.data` also includes creating and reading custom-made datasets.
 To create a dataset, we can do the following:
 
->>> example_hamiltonian = qml.Hamiltonian(coeffs=[1,0.5], observables=[qml.PauliZ(wires=0),qml.PauliX(wires=1)])
->>> example_energies, _ = np.linalg.eigh(qml.matrix(example_hamiltonian)) #Calculate the energies
->>> example_dataset = qml.data.Dataset(data_name = 'Example',hamiltonian=example_hamiltonian,energies=example_energies)
+>>> example_hamiltonian = qml.Hamiltonian(coeffs=[1,0.5], observables=[
+...     qml.PauliZ(wires=0),qml.PauliX(wires=1)])
+>>> example_energies, _ = np.linalg.eigh(
+...     qml.matrix(example_hamiltonian)) #Calculate the energies
+>>> example_dataset = qml.data.Dataset(data_name = 'Example', 
+...                                    hamiltonian=example_hamiltonian,
+...                                    energies=example_energies)
```

Comment:
Same comments here as above:

- make sure we are blacking to ensure consistent formatting
- can reduce variable names? E.g., the `example` is redundant in all cases. We could also use `H` instead of hamiltonian